checador/app/views/companies/index.html.erb

```diff
@@ -1 +1,30 @@
-<h1>Company Branches</h1>
+
+<p id="notice"><%= notice %></p>
+
+<h1>Companies</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Addres</th>
```

Comment:
You also need to update the `Addres` word here

Reply:
Ready Thank you very much we are attentive to any changes